src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1918,12 +1924,21 @@ namespace NServiceBus.Performance.TimeToBeReceived
         public System.TimeSpan MaxTime { get; }
     }
 }
-namespace NServiceBus.Persistence.Legacy
-{
-    public class MsmqPersistence : NServiceBus.Persistence.PersistenceDefinition { }
-}
 namespace NServiceBus.Persistence
 {
+    public interface CompletableSynchronizedStorageSession : NServiceBus.Persistence.SynchronizedStorageSession, System.IDisposable
```

Comment:
What is the use case for the 2 flavors of storage sessions? (when is the completable one used?)